### DISCUSSION TITLE
Is there any specific log write on file??

### DISCUSSION THREAD

**Discussion by Guiwoo** [comfyanonymous/ComfyUI#3211] [Q&A]
Hello i'm a newbie with this git project
Actually i would like to leave all logs as a File 
Would you suggest way or how to do that ? 
Because It's really hard to find error when I run on the container ...

**Comment by reaperhammer** [comment]
On linux, using conda, you can do this;
```conda activate {name of conda env}```
```python ./ComfyUI/main.py 2>&1 | tee -a comfyui_log.txt```
The comfyui_log.txt is written to the current directory, the ```-a``` will append to the existing log, omit this to overwrite log each time you start comfy.

It would be a similar process on windows, modify the .bat file you use to launch comfy, for example if you use ```run_nvidia_gpu.bat```
Modify the line ```.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --port 8189```
To this ```powershell -Command ".\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --port 8189 2>&1 | Tee-Object -FilePath C:\users\{username}\desktop\comfyui_log.txt -Append"```

Modify the path where you want to store the log files accordingly.
